internal/tasks/fleet_validate.go

```diff
@@ -116,7 +116,7 @@ func (t *FleetValidateLogic) validateFleetTemplate(ctx context.Context, fleet *a
 	return retErr
 }
 
-func (t *FleetValidateLogic) validateConfigItem(ctx context.Context, configItem *api.DeviceSpecification_Config_Item) (string, error) {
+func (t *FleetValidateLogic) validateConfigItem(ctx context.Context, configItem *api.DeviceSpec_Config_Item) (string, error) {
```

Comment:
does openAPi generate these names just curious why this is not api.DeviceSpecConfigItem. Is there a config to change that?

Reply:
Yes, the code generator generates that on its own.